packages/contracts/tasks/deploy/maci/09-voiceCreditProxyFactory.ts

```diff
@@ -0,0 +1,86 @@
+import type {
+  ConstantInitialVoiceCreditProxyFactory,
+  ERC20VotesInitialVoiceCreditProxyFactory,
+} from "../../../typechain-types";
+
+import { EDeploySteps } from "../../helpers/constants";
+import { ContractStorage } from "../../helpers/ContractStorage";
+import { Deployment } from "../../helpers/Deployment";
+import {
+  EContracts,
+  EInitialVoiceCreditProxies,
+  EInitialVoiceCreditProxiesFactories,
+  IDeployParams,
+} from "../../helpers/types";
+
+const deployment = Deployment.getInstance();
+const storage = ContractStorage.getInstance();
+
+/**
+ * Deploy step registration and task itself
+ */
+deployment
+  .deployTask(EDeploySteps.InitialVoiceCreditProxyFactory, "Deploy initial voice credit proxy factory")
+  .then((task) =>
+    task.setAction(async (_: IDeployParams, hre) => {
```

Comment:
@0xmad after removing those if blocks you recommended, I no longer needed the `incremental` arg, should I do anything else here?

Reply:
No, let's keep incremental. It's related only to `eslint-disable` comment